### DISCUSSION TITLE
点击过快导致跳过失败

### DISCUSSION THREAD

**Discussion by MCQSJ** [gkd-kit/gkd#770] [Announcements]
### 日志文件

[log-1731284927763.zip](https://github.com/user-attachments/files/17693721/log-1731284927763.zip)

### BUG描述(文字/截图/视频)

打开知乎，腾讯视频偶尔出现提示gkd，但是没有点击成功，看着像是点击太快，并且只会点击一次导致，平时大部分时候是成功的。使用Shizuku保活，设置的Shizuku点击和识别，日志是知乎失败时候的。

### 期望行为(文字/截图/视频)

正常跳过和识别

### 实际行为(文字/截图/视频)

提示gkd但是未跳过
![Screenshot_2024-11-11-08-28-56-519_li songe gkd](https://github.com/user-attachments/assets/eabe3363-e1f9-498b-aea8-eb9cb1e672ea)
图示时间段知乎

**Comment by lisonge** [comment]
你可以设置点击延迟
